### PR TITLE
[REVIEW] Replacing old context API

### PR DESCRIPTION
**Pull request by rommelDB**
This PR is the final port for evaluate_query, here we use the new experimental::Context and experimental::stuff and invoke the new ral compute primitives based on cudf 0.12

### CODE REVIEW

File: engine/src/CalciteInterpreter.cpp [hunk 1]
```diff
@@ -910,7 +909,7 @@ std::unique_ptr<ral::frame::BlazingTable> evaluate_split_query(std::vector<ral::
 //			}
 //		}
 
-//		ral::communication::network::Server::getInstance().deregisterContext(queryContext.getContextToken());
+//		ral::communication::network::experimental::Server::getInstance().deregisterContext(queryContext.getContextToken());
```
Comment (felipeblazing) [inline engine/src/CalciteInterpreter.cpp]: do we no longer need to deregister?

File: engine/src/distribution/primitives.cpp [hunk 2]
```diff
@@ -1108,8 +150,11 @@ void sendSamplesToMaster(Context * context, const BlazingTableView & samples, st
 	auto message =
 		Factory::createSampleToNodeMaster(message_id, context_token, self_node, table_total_rows, samples);
 
+	// // Send message to master
```
Comment (felipeblazing) [inline engine/src/distribution/primitives.cpp]: is this in another pr?

File: engine/src/testing-libgdf.cu [hunk 3]
```diff
@@ -219,8 +219,8 @@ static result_pair systemCommand(uint64_t accessToken, Buffer&& requestPayloadBu
 
       std::thread([]() {
           std::this_thread::sleep_for(std::chrono::seconds(1));
-          ral::communication::network::Client::closeConnections();
-          ral::communication::network::Server::getInstance().close();      
+          ral::communication::network::experimental::Client::closeConnections();
```
Comment (felipeblazing) [inline engine/src/testing-libgdf.cu]: This entire file is no longer used it can be deleted right?

File: engine/src/CalciteInterpreter.cpp (hunk 1 above)
Comment (aucahuasi) [reply engine/src/CalciteInterpreter.cpp]: Don't know I think this was in the legacy code maybe @williamBlazing can answer this

File: engine/src/distribution/primitives.cpp (hunk 2 above)
Comment (aucahuasi) [reply engine/src/distribution/primitives.cpp]: Yep the distribution part seems incomplete so I guess will be addressed in another one

File: engine/src/testing-libgdf.cu (hunk 3 above)
Comment (aucahuasi) [reply engine/src/testing-libgdf.cu]: good catch ... deleted!

File: engine/src/CalciteInterpreter.cpp (hunk 1 above)
Comment (williamBlazing) [reply engine/src/CalciteInterpreter.cpp]: we do want to deregister context

**Pull request merged**